net/miniupnpd/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=miniupnpd
 PKG_VERSION:=2.2.3
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
```

Comment:
```suggestion
+PKG_USE_MIPS16:=0

```

Reply:
PKG_USE_MIPS16:=0 doesn't seem to help :(

Reply:
Disabling LTO for the nftables variant did.